Content.Server/StationEvents/Events/DiseaseOutbreak.cs

```diff
@@ -61,15 +64,24 @@ public override void Startup()
             return;
 
         var diseaseSystem = EntitySystem.Get<DiseaseSystem>();
-        /// Now we give it to people in the list of living disease carriers earlier
+        // Now we give it to people in the list of living disease carriers earlier
+        var _stationSystem = EntitySystem.Get<StationSystem>();
```

Comment:
style guide, also you're resolving IEntitySystemManager twice when you could just resolve it once first for both systems.